app/config/AgentItvcErrorHandler.scala

```diff
@@ -20,15 +20,15 @@ import play.api.i18n.{I18nSupport, MessagesApi}
 import play.api.mvc.Results.{InternalServerError, Ok, Unauthorized}
 import play.api.mvc.{Request, Result}
 import play.twirl.api.Html
-import uk.gov.hmrc.play.bootstrap.frontend.http.FrontendErrorHandler
+import uk.gov.hmrc.play.bootstrap.frontend.http.LegacyFrontendErrorHandler
 import views.html.agent.errorPages.UnauthorisedAgentView
 import views.html.errorPages.templates.ErrorTemplate
 
 import javax.inject.Inject
 
 class AgentItvcErrorHandler @Inject()(val errorTemplate: ErrorTemplate,
                                       val messagesApi: MessagesApi,
-                                      unauthorisedAgentView: UnauthorisedAgentView) extends FrontendErrorHandler with I18nSupport with ShowInternalServerError {
+                                      unauthorisedAgentView: UnauthorisedAgentView) extends LegacyFrontendErrorHandler with I18nSupport with ShowInternalServerError {
```

Comment:
Why we should go back and use LegacyFrontendErrorHandler ? this looks like a step back